src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -392,7 +394,7 @@ def fn(
                     has_other_data = self.check_if_results_have_data(original_results)
                     if isinstance(original_results, SnubaTSResult):
                         dataset_meta = original_results.data.get("meta", {})
-                    else:
+                    elif isinstance(original_results, dict):
```

Comment:
Can we add an else block for `dataset_meta = {}` otherwise, it's possible that `dataset_meta` is undefined.

Reply:
Sure!